### ISSUE TITLE
Failed binding popup

### ISSUE THREAD

**Issue by WSSDude420** [yamashi/CyberEngineTweaks#756]
Show user why binding fails instead of silently rejecting new bind without any reason given.

Idea was to have a popup which would tell user exact reason, and possibly offer solutions.

**Comment by BIGWayne2234** [comment]
well it popped up but i cant click on it
[https://gyazo.com/dfdb5745d9e51f720b769df4b22705d8](url)

**Comment by Floriu** [comment]
i got same problem and i dont know what should i do :(

**Comment by djdomi** [comment]
i agree having issue binding the gui to a key. i click on Bind it say binding and nothing happens...
Is there a manual way to do this?
